test/puppetlabs/ring_middleware/core_test.clj

```diff
@@ -118,6 +146,48 @@
            (add-ring-handler proxy-webserver# ~proxy-handler ~endpoint))
          ~@body)))
 
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;;; Core Helpers
+
+(deftest json-response-test
+  (testing "json response"
+    (let [source {"key1" "val1", "key2" "val2"}
```

Comment:
maps in Clojure are almost always written with a newline between key/value pairs and no comma, e.g.
```clj
{"key1" "val1" 
 "key2" "val2"}
```
also it's highly preferable to use keywords as keys wherever possible.  not sure if that makes sense here or not, being unsure of what `json-response` does exactly.